rust/processor/migrations/2023-08-22-232603_add_ans_view/up.sql

```diff
@@ -0,0 +1,12 @@
+CREATE OR REPLACE VIEW current_ans_lookup_v2 AS
```

Comment:
can you rename this to current_aptos_names or something more generic? Technically it's not v2.